parm/config/gfs/config.base

```diff
@@ -479,4 +479,20 @@ export OFFSET_START_HOUR=0
 # Number of regional collectives to create soundings for
 export NUM_SND_COLLECTIVES=${NUM_SND_COLLECTIVES:-9}
 
+# The tracker, genesis, and METplus jobs are not supported on CSPs yet
+# TODO: we should place these in workflow/hosts/awspw.yaml as part of AWS/AZURE setup, not for general.
+if [[ "${machine}" =~ "PW" ]]; then
+   export DO_TRACKER="NO"
+   export DO_GENESIS="NO"
+   export DO_METP="NO"
+   export DO_WAVE="NO"
+fi
+
+# The tracker and genesis are not installed on Orion/Hercules yet; this requires spack-stack builds of the package.
+# TODO: we should place these in workflow/hosts/[orion|hercules].yaml.
+if [[ "${machine}" == "ORION" || "${machine}" == "HERCULES" ]]; then
+   export DO_TRACKER="NO"
+   export DO_GENESIS="NO"
+fi
```

Comment:
#3053 moved the machine-specific code out of `config.base`. Please do not add it back. Any machine-specific settings should go in the appropriate `workflow/hosts` file.